src/test/java/com/fox/platform/cntsrvex/infra/serv/EndpointVerticleTest.java

```diff
@@ -0,0 +1,65 @@
+package com.fox.platform.cntsrvex.infra.serv;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import com.newrelic.agent.deps.org.apache.http.HttpStatus;
+import io.vertx.core.Vertx;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+
+@RunWith(VertxUnitRunner.class)
+public class EndpointVerticleTest {
+
+
+  private static Logger log = LogManager.getLogger();
+
+  private static Vertx vertx;
+  private int port = Integer.getInteger("http.port", 8080);
+
+   @Before
+  public void setUp(TestContext ctx) {
+    log.info("Iniciando prueba");
+    vertx = Vertx.vertx();
+    vertx.deployVerticle(ProxyChannelsVerticleJuan.class.getName());
+    vertx.deployVerticle(ProxyChannelsVerticle.class.getName());
+    vertx.deployVerticle(EndpointVerticle.class.getName());
+  }
+
+   @After
+  public void down(TestContext ctx) {
+    log.info("Terminando el test");
+    vertx.close(ctx.asyncAssertSuccess());
+  }
+
+   @Test
+  public void testRESTMethodsAle(TestContext ctx) {
+    Async async = ctx.async();
+
+    vertx.createHttpClient().getNow(port, "localhost", "/channels_ale?countryId='CO'", response -> {
+      ctx.assertEquals(response.statusCode(), HttpStatus.SC_OK);
+      response.bodyHandler(body -> {
+        ctx.assertEquals(body.toJsonArray().size(), 4);
```

Comment:
Validar este javadoc para cada metodo de la prueba 	
/**
	 * Integration test with the Scenario
	 * SCENARIO: UpdateLevelById
	 * GIVEN a mediaID valid
	 * AND   a authlevel valid 
	 * WHEN execute the method onMessage(Verticle) 
	 * THEN Return a UpdateLevelResponse not NULL
	 * 1. test that the response is NOT NULL
	 * 2. Test that the streamID is NOT NULL.
	 * 3. Test that the authLevelOld is NOT NULL.
	 * @param context
	 */

Reply:
Se usó el formato sugerido